conf/seattlechildrens.config

```
@@ -0,0 +1,29 @@
+//Create profiles to easily switch between the different process executors and platforms.
+
+//global parameters
+params {
+    config_profile_description = 'The SCRI (seattle childrens research institute) cluster profile'
+    config_profile_contact = 'Research Scientific Computing (@RSC-RP)'
+    config_profile_url = 'https://github.com/RSC-RP'
+
+    // SCRI HPC project params
+    queue                       = "paidq" // freeq
+    project                     = ""      // please put your own project code 
```

The config file is not editable by the user, so maybe include a param here

```suggestion
    project                     = "${params.project}" 
```